askap/imageaccess/ADIOSImage.tcc

```diff
@@ -95,16 +99,29 @@ void ADIOSImage<T>::makeNewTable(const casacore::TiledShape& shape, casacore::uI
   const casacore::uInt ndim = latShape.nelements();
 
   casacore::TableDesc description;
+  //PJE - why are we adding a hard-coded string???
```

Comment:
This was a quick thing to make sure it works. Ideally, this will be passed via the config file or as an input parameter.